src/pybind/mgr/dashboard/frontend/src/app/shared/pipes/cd-date.pipe.ts

```diff
@@ -0,0 +1,17 @@
+import { DatePipe } from '@angular/common';
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'cdDate'
```

Comment:
All current existing pipes are not using the prefix ``cd``, so i suggest to keep this and remove it in this PR too.

Reply:
@votdev But angular already have a pipe named `date`, it will cause conflicts.

Reply:
Sounds good to me.

Reply:
@votdev @tspmelo per our discussion today, we will keep the name `cdDate`.